resources/mysqlbinlog/mysqlbinlog_8.0.28-macos11-arm64_test.go

```diff
@@ -0,0 +1,43 @@
+//go:build darwin && arm64
+// +build darwin,arm64
+
+package mysqlbinlog
+
+import (
+	"embed"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/bytebase/bytebase/resources/utils"
+)
+
+//go:embed mysqlbinlog-8.0.28-macos11-arm64.tar.gz
+var resources embed.FS
+
+// TestBootMySQLBinlog tests whether mysqlbinlog can be started on the target platform
+// to check whether the lib extraction is correct.
+func TestBootMySQLBinlog(t *testing.T) {
+	tarName := "mysqlbinlog-8.0.28-macos11-arm64.tar.gz"
+	version := "mysqlbinlog-8.0.28-macos11-arm64"
+	tarF, err := resources.Open(tarName)
+	if err != nil {
+		t.Errorf("failed to open mysqlbinlog dist %q, error: %v", tarName, err)
+	}
+	defer tarF.Close()
+
+	tmpDir := t.TempDir()
+	if err := utils.ExtractTarGz(tarF, tmpDir); err != nil {
+		t.Errorf("failed to extract mysqlbinlog distribution %q, error: %v", tarName, err)
+	}
+
+	mysqlbinlogBinPath := filepath.Join(tmpDir, version, "bin", "mysqlbinlog")
```

Comment:
```suggestion
	mysqlbinlogPath := filepath.Join(tmpDir, version, "bin", "mysqlbinlog")
```

Reply:
Sorry for the nitpicking, just so much "bin" here 😂

Reply:
That's ok, lol. I just want to clarify that there is a bin directory. Follow you.